rllib/models/catalog.py

```diff
@@ -50,268 +50,69 @@
 # fmt: off
 # __sphinx_doc_begin__
 MODEL_DEFAULTS: ModelConfigDict = {
```

Comment:
`DefaultModelConfig.to_dict()`?

Reply:
Let's keep it as-is (will be gone, soon, anyways). Some names/keys have changed, so this would be a nightmare.